es-common-extensions/src/debug-metrics.tsx

```diff
@@ -1,4 +1,4 @@
-import type { ExtensionScaffoldApi } from "@moesol/es-runtime/build/es-api"
+import type { ExtensionScaffoldApi } from "@moesol/es-runtime/src/es-api"
```

Comment:
I encountered issues with Vite's bundling process which overly aggressively aggregates modules even when configurations like minify: false and preserveEntrySignatures: true are employed. This aggressive bundling disrupts the integrity of the code when executed on the frontend. As a workaround, I've opted to directly import TypeScript files from the /src directory, bypassing Vite’s bundling mechanisms. However, this approach introduces compatibility issues with JavaScript-only environments, as these cannot natively interpret TypeScript files. Additionally, this direct import strategy from source without compilation is suboptimal for production readiness. We need a more robust solution that maintains module fidelity while ensuring broad compatibility across environments.